src/multi-level-checkbox-editor/multi-level-checkbox-editor.tsx

```diff
@@ -104,6 +115,10 @@ class MultiLevelCheckboxEditor extends React.Component<Props, State> {
         );
     }
 
+    private isEmptyBox() {
+        return this.props.maxBoxCount && this.state.selectedNodeIds.length + 1 < this.props.maxBoxCount;
```

Comment:
I don't really understand the meaning of Empty Box, why would you call something empty if it returns a value?
imagine `maxBoxCount = 6`
`selectedNodeIds.length = 2` 
then `2 + 1 < 6 ` returns `true` which represented as empty.

Maybe I miss understand something, can you explain to me?

Reply:
yeah true the name should be changed thanks :)